pkg/cli/zip_test.go

```diff
@@ -128,9 +129,9 @@ func TestZip(t *testing.T) {
 	// Strip any non-deterministic messages.
 	out = eraseNonDeterministicZipOutput(out)
 
-	datadriven.RunTest(t, "testdata/zip/testzip", func(t *testing.T, td *datadriven.TestData) string {
```

Comment:
Please, no. The purpose of the datadriven calls was to enable TESTFLAGS=-rewrite. Your change breaks that.

Reply:
oh -rewrite, interesting. I've put that in the comment below and got rid of this commit.